.github/workflows/pull-request.yml

```
@@ -91,538 +48,356 @@
-            echo "Apollo server is down"
-            exit 1
-          fi
-          echo "Stopping development server..."
-          kill -9 $DEV_PID || echo "Process already stopped."
+          # Count all changed files
+          ALL_CHANGED_FILES_COUNT=$(git diff --name-only --diff-filter=ACMRT $BASE_SHA ${{ github.event.pull_request.head.sha }} | wc -l | tr -d ' ')
+          echo "all_changed_files_count=$ALL_CHANGED_FILES_COUNT" >> $GITHUB_OUTPUT
 
-      - name: Generate SSL Certificates for Production
+      - name: Echo number of changed files
+        env: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)